frontends/user-articles/components/PublishArticleModal.tsx

```diff
@@ -76,12 +76,18 @@ export const PublishArticleModal: FC<{
     if (props.open) publishBtnRef.current?.focus();
   }, [props.open]);
 
+  console.log(theme);
```

Comment:
remove this console

Reply:
Ooh, it was hard to create webp animation with transparent background. Thatʼs why I used apng format